app/views/admin/users/index/_user_status_indicator.html.erb

```diff
@@ -1,5 +1,7 @@
 <% if user.suspended? %>
     <%= render "admin/users/index/status_indicator", status: "Suspended" %>
+<% elsif user.spam? %>
+    <%= render "admin/users/index/status_indicator", status: "Spam" %>
```

Comment:
I don't get why `span` is empty here:
https://github.com/forem/forem/blob/main/app/views/admin/users/index/_status_indicator.html.erb

So, locally changing bg color doesn't change anything, but I have checked git history and it was like this initially :shrug: , does it look different on prod?

Reply:
I'm not sure either but it looks like it's been this way since the beginning